### PR TITLE
rubocop configuration revisit, mostly auto-fixes

### PR DESCRIPTION
**Pull request by SlakrHakr**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -43,29 +43,6 @@ Rails/I18nLocaleTexts:
...
-#   ...
-#   if test
-#     ...
-#   end
-# end
-#
-# would be considered bad, and the cop would force you to put a `return if !test` before that block and
-# then remove the if. The problem is that this hides intent, since the if test does have a purpose in
-# readability, and it could also be easier for future changes to miss the return statement and add code
-# after it expecting it to be executed.
```
Comment (SlakrHakr) [inline .rubocop.yml:59]: i disagree with this statement

File: .rubocop.yml [hunk 2]
```diff
@@ -43,29 +43,6 @@ Rails/I18nLocaleTexts:
...
-Style/GuardClause:
-  Enabled: false
-
-# This is pretty much the same thing as the one above. Inside a loop, it forces you to use next to skip
-# iteration rather than using an if block that runs to the end of the loop, and it suffers from the same
-# problems as above.
```
Comment (SlakrHakr) [inline .rubocop.yml:65]: same as above

File: Gemfile [hunk 3]
```diff
@@ -39,7 +39,6 @@ gem 'rest-client'
 gem 'rufus-scheduler'
 gem 'sass-rails', '>= 6'
 gem 'serviceworker-rails'
-gem 'simple_form'
```
Comment (SlakrHakr) [inline Gemfile:42]: i dont think this is needed anymore? for now at least

Comment (FonzMP) [reply Gemfile:42]: yeah, this was an old gem from the UI days I believe.

**Review by FonzMP — APPROVED** [review]
:shipit:

**Comment by FonzMP** [comment]
Oh yeah, the good stuff.....
```
15:41:05  Line coverage by file (88.88%) is below the expected minimum coverage (90.00%).
15:41:05  "./app/api/v0/entities/legacy/api_provider_entity.rb",89,81,45,40,5
```

**Comment by SlakrHakr** [comment]
im gonna revisit this later, already have all those other changes going in from yesterday

**Pull request closed without merging**